### DISCUSSION TITLE
Certain Quibbles

### DISCUSSION THREAD

**Discussion by aMediocreDad** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#34] [Ideas]
As I have been working on the content modules I have encountered things that aren't supported by the system or thing that work contrary to how I expect it. These are usually minor things, but some require larger changes.

I'll put it in a checked list so I can keep track of it. But I'd also like to discuss how or if even we should tackle these issues.

- [ ] Monster Talents are largely redundant. These could be merged with normal Talents. Esepcially now that we can add a talent type to indicate that it is a monster talent.
- [ ] Monster spells. The monster sheet should support spells. 
- [ ] Monster attacks aren't numbered and rollable.
- [ ] Talents Ranks as a dropdown menu? (Or maybe split talents into three boxes if they are profession/general talents and put a rank dropdown in the character sheet?)
- [ ] Too much white space in main tab of character sheet. 
- [x] Support for heavier weight items (numbers).
- [ ] Animal category for items? For mounts and pets (instead of giving them a full character sheet).
- [ ] Is it necessary to expose the World Version to users? I do not see any use in it beyond convenience when developing something that requires a manifest update.

---

... More to come as they come to mind.

**Comment by jimorie** [comment]
I think all of these sound like reasonable quibbles to have, and if someone wants to put the time in we should fix them.

> Monster Talents are largely redundant. 

As long as we migrate existing monster talents into regular talents that should be fine, I think.

> Too much white space in main tab of character sheet.

I guess for the issue with the main tab of the character sheet having too much white space we should ask ourselves what the long term plan for the sheets are. Are we thinking about a major re-design, then maybe it isn't worth spending too much time on the current one?

> Support for heavier weight items (numbers).

Maybe the easy way is to just add a couple of more weight categories? E.g. "Very heavy (x3)", "Super-duper heavy (x4)", or similar. I'm just thinking that adding another input field for a custom weight would clutter the UI more than it'd be worth.

**Comment by Kalima** [comment]
> Animal category for items? For mounts and pets (instead of giving them a full character sheet).

I would prefer if mounts had their own character sheet; a group of players can share a mount as a beast of burden, and having it attached to one particular character can make this cumbersome. On the other hand, pets are almost always attached to a specific character.

An option would be to make it possible to drag and drop a mount or a pet character sheet in an "Extras" tab on the players' characters' sheets, to link them without having to treat mounts and pets as items (which makes little sense mechanically since they have different attributes and features than pure items).

**Reply by aMediocreDad** [reply]
Please open an issue on this. This is something we intend to implement at some point as part of a character/monster sheet redesign.

**Reply by Kalima** [reply]
Will do :)
